samples/multimodality/xDiT/README.md

```diff
@@ -0,0 +1,182 @@
+# Deploying AIBrix with xDiT
+## Prerequisites
+### Download xDiT and patch to xDiT
+1. Download patch from [xDiT-intergation/xdit-52e74e88d2332281eefe68894af02f805a1d2b4f.patch](xDiT-intergation/xdit-52e74e88d2332281eefe68894af02f805a1d2b4f.patch). The following steps assume the patch will be in your current path and apply this patch automatically while using `Dockerfile`. 
+2. Download xDiT source code from [xdit-project/xDiT](https://github.com/xdit-project/xDiT) and copy patch file.
```

Comment:
could you pin the version? the main branch alway changes and there's no guarantee the patch will work all the time

Reply:
I see `52e74e88d2332281eefe68894af02f805a1d2b4f`, great